modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/DistributionZoneSqlDdlParserTest.java

```diff
@@ -179,6 +179,33 @@ public void alterZoneIfExistsRenameTo() {
         expectUnparsed(alterZone, expectedStmt);
     }
 
+    /**
+     * Parsing ALTER ZONE SET DEFAULT statement.
+     */
+    @Test
+    public void alterZoneSetDefault() {
```

Comment:
let's add test for negative scenario where within a single command we change any attribute of the zone and set it as default

Reply:
Added a separate test `alterZoneSetDefaultWithOptionsIsIllegal`